client/src/components/LoginDialog.tsx

```diff
@@ -37,6 +39,13 @@ const Title = styled.h1`
   text-align: center;
 `
 
+const SubTitle = styled.h1`
+  width: 160px;
+  font-size: 16px;
+  color: #eee;
```

Comment:
optional nit: I think you don't need this anymore, it inherits from the outer color already.

Reply:
I still need this somehow. Otherwise, the text would be black.